build/UIKit/lib/UIKitLib.vcxproj

```diff
@@ -110,6 +110,20 @@
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIView.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIViewController.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIWindow.mm" />
+    <ClangCompile Include="..\..\..\Frameworks\UIKit\WYPopoverController.m">
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">true</ObjectiveCARC>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Disabled</OptimizationLevel>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">Disabled</OptimizationLevel>
```

Comment:
Disabled optimization to address preferredContentSize issue? This is the part that I don't really like - not understanding the root cause. I'll let someone else be the tie breaker.